src/WebUI/Views/Shared/_Layout.cshtml

```diff
@@ -1,48 +1,52 @@
 ﻿<!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>@ViewData["Title"]</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <script src="~/lib/jquery/dist/jquery.min.js"></script>
```

Comment:
Note, that scripts should be loaded asynchronous or deferred according to the coding standards :
https://github.com/codidact/core/wiki/Code-standards:-HTML#25-links-and-scripts